Tools/TestWebKitAPI/Tests/WebKit Swift/WebPage.swift

```diff
@@ -189,6 +190,30 @@ struct WebPageNavigationTests {
 
         #expect(actualEventKinds == expectedEventKinds)
     }
+
+    @Test
+    func observableProperties() async throws {
+        let page = WebPage_v0()
+
+        let html = """
+        <html>
+        <head>
+            <title>Title</title>
+        </head>
+        <body></body>
+        </html>
+        """
+
+        #expect(page.url == nil)
+        #expect(page.title == "")
+        #expect(!page.isLoading)
+        #expect(page.estimatedProgress == 0.0)
+        #expect(page.serverTrust == nil)
+        #expect(!page.hasOnlySecureContent)
+        #expect(page.themeColor == nil)
+
+        // FIXME: Make this test more comprehensive once Observation supports observing a stream of changes to properties.
```

Comment:
Do you have a bug for this?

Reply:
fixed; I added a reference to the bug in the comment